packages/data-stores/src/queries/use-launchpad.ts

```diff
@@ -0,0 +1,98 @@
+import apiFetch from '@wordpress/api-fetch';
+import { useQuery } from 'react-query';
+import wpcomRequest, { canAccessWpcomApis } from 'wpcom-proxy-request';
+
+interface APIFetchOptions {
+	global: boolean;
+	path: string;
+}
+
+interface Task {
+	id?: string;
```

Comment:
The `id`, `completed`, `disabled`, and `title` attributes aren't optional

Reply:
Updated these to be non-optional.